src/Extension/StatusRuntime.php

```diff
@@ -43,14 +44,18 @@ public function addStatusService(StatusClassRendererInterface $renderer): void
      */
     public function statusClass($object, $statusType = null, string $default = ''): string
     {
-        if (\is_object($object)) {
+        if (\is_object($object) && !($object instanceof FlashManagerInterface)) {
             return $this->statusClassForStatusClassRenderer($object, $statusType, $default);
         }
 
         if (\is_string($object)) {
             return $this->statusClassForFlashManager($object, $statusType, $default);
         }
 
+        if ($object instanceof FlashManagerInterface) {
```

Comment:
If you check this first, you'll avoid the need to check `&& !($object instanceof FlashManagerInterface)` in the first `if`.